vendor/bat-native-ledger/src/bat/ledger/internal/database/migration/migration_v34.h

```diff
@@ -0,0 +1,33 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_VENDOR_BAT_NATIVE_LEDGER_SRC_BAT_LEDGER_INTERNAL_DATABASE_MIGRATION_MIGRATION_V34_H_
+#define BRAVE_VENDOR_BAT_NATIVE_LEDGER_SRC_BAT_LEDGER_INTERNAL_DATABASE_MIGRATION_MIGRATION_V34_H_
+
+namespace ledger {
+namespace database {
+namespace migration {
+
+// Migration 34 attempts to restart failed ACs where BAT was transferred from a
+// user's external wallet but the SKU transaction was never completed.
+const char v34[] = R"sql(
```

Comment:
Do we want to add a test case for that in `LedgerDatabaseMigrationTest`?

Reply:
Yep - we probably should.